web/core/src/jsMain/kotlin/androidx/compose/web/attributes/InputAttrsBuilder.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020-2021 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package androidx.compose.web.attributes
+
+import org.jetbrains.compose.web.attributes.*
+import org.jetbrains.compose.web.events.GenericWrappedEvent
+import org.jetbrains.compose.web.events.WrappedCheckBoxInputEvent
+import org.jetbrains.compose.web.events.WrappedRadioInputEvent
+import org.jetbrains.compose.web.events.WrappedTextInputEvent
+import org.w3c.dom.HTMLElement
+import org.w3c.dom.HTMLInputElement
+import org.w3c.dom.events.Event
+
+class SyntheticInputEvent<ValueType, Element : HTMLElement>(
+    val value: ValueType,
+    val target: Element,
+    val nativeEvent: Event
+)
+
+class InputAttrsBuilder<T>(val inputType: InputType<T>) : AttrsBuilder<HTMLInputElement>() {
+
+    fun onInput(options: Options = Options.DEFAULT, f: (SyntheticInputEvent<T, HTMLInputElement>) -> Unit) {
+        addEventListener(INPUT, options) {
+            val value = inputType.inputValue(it.nativeEvent)
+            f(SyntheticInputEvent(value, it.nativeEvent.target as HTMLInputElement, it.nativeEvent))
+        }
+    }
+
+    @Deprecated(
```

Comment:
When we will remove deprecated API ?

Reply:
I think we'll remove it in a couple of weeks.